src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -233,19 +234,86 @@ def load_command_table(self, _):
         g.command('list', 'list')
         g.custom_show_command('show', 'get_deployment_operations', client_factory=cf_deployment_operations)
 
+    # az deployment
     with self.command_group('deployment', resource_deployment_sdk, min_api='2018-05-01', resource_type=ResourceType.MGMT_RESOURCE_RESOURCES) as g:
```

Comment:
Please make sure the min_api constraint is right for your new commands.

Reply:
yes, this is still the right min_api.